e2e/fixtures/recitals.js

```diff
@@ -0,0 +1,4 @@
+module.exports = [{
```

Comment:
Are we likely to add more than one? If not i'd remove this file and add title & description directly.

Reply:
removing